service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -115,13 +118,103 @@ public class EhrExtractStatusService {
     private static final String FILENAME_TYPE_PLACEHOLDER = "FILENAME_PLACEHOLDER_ID=";
     private static final String ACKS_SET = ACK_HISTORY + DOT + ACKS;
     public static final int EHR_EXTRACT_SENT_DAYS_LIMIT = 8;
-    private static final String REASON_ERROR_CODE = "04";
+    private static final String REASON_ERROR_CODE = "99";
```

Comment:
Out of scope for this PR possibly, though wouldn't it be better to separate this out into an enumeration of sorts? I could see the benefit of having a central location for these error codes I see in this codebase.

Furthermore, a little comment to describe what these mean could be useful! Not a clue what `99` means to us in this context! 😄

Reply:
I think it is a good point for the future refactoring effort and I see the benefit in aggregating all the error codes as Enums in one place 👍. Will take a note of it. The error code will be renamed to be more descriptive. 👍